cli/cmd/play.go

```diff
@@ -118,10 +123,17 @@ func internalPlayModule(cmdCtx *cmdcontext.CmdCtx, args []string) error {
 	os.Setenv("TT_CLI_PLAY_FROM", strconv.FormatUint(playFlags.From, 10))
 	os.Setenv("TT_CLI_PLAY_TO", strconv.FormatUint(playFlags.To, 10))
 
+	iProtoTimestamp, err := util.TsStrToIprotoTs(playFlags.Timestamp)
```

Comment:
IPROTO_TIMESTAMP is confusing name. It think it could be something like this:
```suggestion
	timestamp, err := util.StringToTimestamp(playFlags.Timestamp)
```